selfdrive/car/toyota/interface.py

```diff
@@ -195,20 +185,15 @@ def get_params(candidate, fingerprint, vin=""):
     # to a negative value, so it won't matter.
     ret.minEnableSpeed = -1. if (stop_and_go or ret.enableGasInterceptor) else 19. * CV.MPH_TO_MS
 
-    centerToRear = ret.wheelbase - ret.centerToFront
     # TODO: get actual value, for now starting with reasonable value for
     # civic and scaling by mass and wheelbase
-    ret.rotationalInertia = rotationalInertia_civic * \
-                            ret.mass * ret.wheelbase**2 / (mass_civic * wheelbase_civic**2)
+    ret.rotationalInertia = scale_rot_inertia(mass=ret.mass, wheelbase=ret.wheelbase)
```

Comment:
style preference: no need to repeat the input variable name. Since the inputs are non-optional, I would prefer:
```
scale_rot_inertia(ret.mass, ret.wheelbase)
```